lang/src/org/partiql/lang/ast/passes/V0AstSserializer.kt

```diff
@@ -106,6 +106,9 @@ class V0AstSerializer(private val writerContext: IonWriterContext) {
                         if(metas.hasMeta(IsCountStarMeta.TAG)) {
                             symbol("call_agg_wildcard")
                             symbol("count")
+                            if(setQuantifier == SetQuantifier.DISTINCT) {
+                                symbol(setQuantifier.toString().toLowerCase())
+                            }
```

Comment:
Same as with the `AstSerializer` class.

Also FYI the V0 `AstSerializer` is being dropped in #154.